app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :comments_counter
+  after_destroy :comments_counter
+
+  def comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `comments_counter` method every time a new comment has been saved to the database, then make the `comments_counter` method private

Reply:
Thank you @DeliceLydia